src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $white-color;
```

Comment:
The border color variable `$white-color` should be defined in your variables file. Ensure that `$white-color` is correctly declared and imported.